internal/server/proto/server.proto

```diff
@@ -2387,6 +2387,9 @@ message StatusReport {
     // Resource id
     string id = 4;
   }
+
+  // the time when this report was generated
+  google.protobuf.Timestamp time_generated = 10;
```

Comment:
Small consistency thing: elsewhere in the schema we seem to use `_time` as a suffix rather than a prefix. i.e. `Workspace.active_time`, `Status.start_time`, `Status.complete_time`, `Job.complete_time`. I wonder if it might be worth following suit here, even if it doesn’t match the SDK quite as well.

```suggestion
  google.protobuf.Timestamp generated_time = 10;
```